packages/core/src/processors/memory/working-memory.ts

```
@@ -159,10 +165,21 @@ export class WorkingMemory implements Processor {
     template: WorkingMemoryTemplate;
     data: string | null;
+    toolInstruction?: string;
   }): string {
+    if (toolInstruction) {
+      return `${toolInstruction}
+
+<working_memory_data>
+${data}
+</working_memory_data>`;
+
+      return `${toolInstruction}\n<working_memory_data>\n${data}\n</working_memory_data>`;
```

Biome (2.4.7)

[error] This code will never be reached ...

(lint/correctness/noUnreachable)